### DISCUSSION TITLE
klipperscreen dialog/popups to big

### DISCUSSION THREAD

**Discussion by GTreaper** [bassamanator/Sovol-SV06-firmware#128] [General]
at this point any help is welcome
hopefully someone knows hot to fix this

**Describe the bug**
Klipperscreen pop up/dialog windows to big for screen.

if i press save config or update the window is to big for the screen so i cant accept or decline and the screen is stuck on this

![293397526-3902c91f-7402-4b79-90b6-1ff32ddeab77](https://github.com/bassamanator/Sovol-SV06-firmware/assets/99057390/a0357de4-113a-44d0-bd93-7a2f138e2921)

**What printer are you using?**

SV06 Plus

**What kind of device are you using to host Klipper?**

Raspberry Pi 4
**How did you install Klipper?**

KIAUH

**Are you using the config bundles from this repo?**
Yes

ive tried changing settings in rasp config.txt tried different framebuffer etc

have a thread on klipperscreen aswell https://github.com/KlipperScreen/KlipperScreen/discussions/1220

klipperscreen log:

[KlipperScreen (1).log](https://github.com/bassamanator/Sovol-SV06-firmware/files/13806144/KlipperScreen.1.log)

**Comment by bassamanator** [comment]
Nothing to do with this config.

My guess is that Fluidd is not detecting the viewport correctly; https://www.w3schools.com/css/css_rwd_viewport.asp.

**Reply by GTreaper** [reply]
Ok could you please explain?
This isn't a fluidd issue, the picture is an example of when I try to update fluidd from the klipperscreen. So when I press update on the klipperscreen this window off accept / decline on klipperscreen opens and is to big and goes outside the screen.

This happens whenever there is a popup/ accept-decline window in klipperscreen
Save config and so on 

Check this video 

https://github.com/bassamanator/Sovol-SV06-firmware/assets/99057390/400436a8-ba95-490b-a3d6-8732d8cc4ac4

**Comment by GTreaper** [comment]
UPDATE: After many changes i finally cracked the nut..
The issue relates to the raspb boot option, booting to desktop environment is whats causing this.
Booting to console fixed the issue.

**Comment by bassamanator** [comment]
Great!
